install.ps1

```diff
@@ -0,0 +1,91 @@
+$ProgressPreference = 'SilentlyContinue'
+$ErrorActionPreference = "Stop"
+$InstallationDirectory = $PSScriptRoot
+
+#
+# Assert required variables are set
+#
+if (Test-Path env:SemaphoreEndpoint) {
+  $SemaphoreEndpoint = $env:SemaphoreEndpoint
+} else {
+  if (-not (Test-Path env:SemaphoreOrganization)) {
+    Write-Warning 'Either $env:SemaphoreOrganization or $env:SemaphoreEndpoint needs to be specified. Exiting...'
+    Exit 1
+  }
+
+  $SemaphoreEndpoint = "$env:SemaphoreOrganization.semaphoreci.com"
```

Comment:
This hardcodes `.semaphoreci.com` domain.
Would be nice to support PreProd and OnPrem versions as well.

Reply:
I just read the next line 🙈 
Everything is 🆗